README.md

```diff
@@ -139,7 +139,7 @@ Similar to `git log`, the `commits` table includes all commits in the history of
 
 | Column          | Type     |
 |-----------------|----------|
-| id              | TEXT     |
+| commit_id       | TEXT     |
```

Comment:
let's keep this just `id` since it's implied that it's a commit ID from the table name